### ISSUE TITLE
[webAPI] Feature Policy now supports `speaker`

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#7746]
[Evaluate Feature policy allow/block "speaker-selection" in WebRTC API](https://bugzilla.mozilla.org/show_bug.cgi?id=1577199)

## Acceptance Criteria

- [ ] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [ ] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [ ] Add entry to Firefox release notes if feature is enabled in release
or
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

- Selecting a speaker is part of the permissions policy. This affects selectAudioOutput(), setSinkId() and enumerateDevices(). Worth checking webRTC docs as well.

## Related Gecko bugs

- [SHA 256 Digest Authentication](https://bugzilla.mozilla.org/show_bug.cgi?id=472823). Looks like a header addition of some kind, but needs investigation.

**Comment by hamishwillee** [comment]
@Rumyra The *Related Gecko bugs* "SHA 256 Digest Authentication" does not match the heading, while up the top you have listed a bug "Evaluate Feature policy allow/block "speaker-selection" in WebRTC API". Is the one under gecko a typo - note, it says that it is related to FF93. I will proceed as if they are unrelated at this point.

**Comment by hamishwillee** [comment]
Status
- [ ] Remove from Experimental Features page - NA as is not present (`dom.security.featurePolicy.experimental.enabled`). Note, that feature policy is disabled by default entirely (`dom.security.featurePolicy.header.enabled`).
- [ ] Docs - Release notes
- [ ] Docs: [Header > Feature-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy) - add `speaker-selection` directive with default value of `self`
- [ ] Docs: [HTTP > Feature Policy > Granular control over certain feaures](https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy#granular_control_over_certain_features) - add speaker to list
- There is talk of [MediaDevices.enumerateDevice()](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices) and  `MediaDevices.selectAudioOutput()` (undocumented). May need to add info to these.
- [ ] BCD - add `speaker-selection` permission for feature policy and permission policy

**Comment by Rumyra** [comment]
> @Rumyra The _Related Gecko bugs_ "SHA 256 Digest Authentication" does not match the heading, while up the top you have listed a bug "Evaluate Feature policy allow/block "speaker-selection" in WebRTC API". Is the one under gecko a typo - note, it says that it is related to FF93. I will proceed as if they are unrelated at this point.

Yes apologies, that fell through a copy and paste 🤷🏻‍♀️ - I've removed 👍

**Comment by hamishwillee** [comment]
This is pretty much done, except for the excitement of BCD review
